src/appmixer/intelswift/core/FindMessage/FindMessage.js

```
@@ -0,0 +1,21 @@
+'use strict';
+
+const { makeRequest } = require('../../commons');
+module.exports = {
+
+    async receive(context) {
+
+        const { tenantId, projectUUID, lastDays = 1, properties = "conversationId, query, reply"  } = context.messages.in.content;
+
+        let body = {};
```

ESLint

[error] 'body' is assigned a value but never used.

(no-unused-vars)